<!DOCTYPE html>
<html>
<head>
  <title>GSCA: Gene Set Context Analysis</title>
  <meta charset="utf-8">
  <meta name="description" content="GSCA: Gene Set Context Analysis">
  <meta name="author" content="Zhicheng Ji, Hongkai Ji">
  <meta name="generator" content="slidify" />
  <meta name="apple-mobile-web-app-capable" content="yes">
  <meta http-equiv="X-UA-Compatible" content="chrome=1">
  <link rel="stylesheet" href="libraries/frameworks/io2012/css/default.css" media="all" >
  <link rel="stylesheet" href="libraries/frameworks/io2012/css/phone.css" 
    media="only screen and (max-device-width: 480px)" >
  <link rel="stylesheet" href="libraries/frameworks/io2012/css/slidify.css" >
  <link rel="stylesheet" href="libraries/highlighters/highlight.js/css/tomorrow.css" />
  <base target="_blank"> <!-- This amazingness opens all links in a new tab. -->  <link rel=stylesheet href="./assets/css/ribbons.css"></link>

  
  <!-- Grab CDN jQuery, fall back to local if offline -->
  <script src="http://ajax.aspnetcdn.com/ajax/jQuery/jquery-1.7.min.js"></script>
  <script>window.jQuery || document.write('<script src="libraries/widgets/quiz/js/jquery.js"><\/script>')</script> 
  <script data-main="libraries/frameworks/io2012/js/slides" 
    src="libraries/frameworks/io2012/js/require-1.0.8.min.js">
  </script>
  
  

</head>
<body style="opacity: 0">
  <slides class="layout-widescreen">
    
    <!-- LOGO SLIDE -->
        <slide class="title-slide segue nobackground">
  <aside class="gdbar">
    <img src="assets/img/bloomberg_shield.png">
  </aside>
  <hgroup class="auto-fadein">
    <h1>GSCA: Gene Set Context Analysis</h1>
    <h2>A Software Tool for Turning Publicly Available Gene Expression Data into Discoveries</h2>
    <p>Zhicheng Ji, Hongkai Ji<br/></p>
  </hgroup>
    <a href="https://github.com/zji90/GSCA-presentation/zipball/gh-pages" class="example">
     Download
    </a>
  <article></article>  
</slide>
    

    <!-- SLIDES -->
    <slide class="" id="slide-1" style="background:;">
  <hgroup>
    <h2>GSCA Overview</h2>
  </hgroup>
  <article data-timings="">
    <p><br></p>

<ul>
<li><p>An open source software package with a powerful Graphical User Interface (GUI).</p></li>
<li><p>Interactively visualize and explore gene and gene set activities in 25,000+ human and mouse microarray samples.</p></li>
<li><p>Systematically identify biological contexts associated with specific gene set activity patterns. </p></li>
<li><p>Discover previously unknown contexts of gene set functions.</p></li>
</ul>

  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="" id="slide-2" style="background:;">
  <hgroup>
    <h2>Background</h2>
  </hgroup>
  <article data-timings="">
    <ul>
<li><p>200,000+ gene expression samples stored in public databases such as GEO and ArrayExpress.</p></li>
<li><p>PED contain vast amount of information, but not fully utilized so far.</p></li>
</ul>

<!-- Microarray dataset 1 generated by one investigator for studying pathway 2 may also contain information about pathway 1. This information may not be used by the original investigator for his/her study of pathway 2, but it can be useful for other people who want to study pathway 1 --> 

<p><center><img src="fig/background.png" width="70%"></center></p>

  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="" id="slide-3" style="background:;">
  <hgroup>
    <h2>Challenges</h2>
  </hgroup>
  <article data-timings="">
    <p><br></p>

<ul>
<li><p>Reuse of PED enables one to systematically examine gene or pathwayâ€™s activities in a broad spectrum of biological contexts, saving resources of generating similar data.</p></li>
<li><p>Several roadblocks to be removed before PED can be reused:</p>

<ul>
<li>Gene expression data should be constantly normalized and annotated.</li>
<li>Data should be easy to visualize and retrieve.</li>
<li>One needs efficient way to analyze the data and turn them into useful biological knowledge.</li>
</ul></li>
<li><p>None of these is trivial given the complexity, heterogeneity and size of the data.</p></li>
</ul>

  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="" id="slide-4" style="background:;">
  <hgroup>
    <h2>ChIP-PED</h2>
  </hgroup>
  <article data-timings="">
    <ul>
<li><p>Analyzes the expression of a transcription factor (TF) and its target genes in PED to find biological contexts associated with the function of the TF.</p></li>
<li><p>Myc: a regulator gene that codes for a transcription factor.</p></li>
</ul>

<p><center><img src="fig/chipped1.png" width="75%"></center></p>

  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="" id="slide-5" style="background:;">
  <hgroup>
    <h2>ChIP-PED</h2>
  </hgroup>
  <article data-timings="">
    <ul>
<li>Knockdown of MYC: substantially slower proliferation rate and decreased tumorigenicity.</li>
</ul>

<p><center><img src="fig/chipped2.png" width="100%"></center></p>

  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="" id="slide-6" style="background:;">
  <hgroup>
    <h2>ChIP-PED Limitations</h2>
  </hgroup>
  <article data-timings="">
    <p><br></p>

<ul>
<li><p>Can only analyze one TF and its target genes.</p></li>
<li><p>The gene expression pattern is basic and limited (TF+TG+; TF+TG-; TF-TG+; TF-TG-)</p></li>
<li><p>Only command-lines. Hard to use.</p></li>
</ul>

  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="" id="slide-7" style="background:;">
  <hgroup>
    <h2>GSCA</h2>
  </hgroup>
  <article data-timings="">
    <p><br></p>

<ul>
<li><p>Built upon 25,000+ pre-normalized and expert-annotated human and mouse gene expression samples with 1000+ biological contexts.</p></li>
<li><p>Takes any number of gene sets of any kind as inputs (ChIP-PED: one TF and its target genes).</p></li>
<li><p>Allows users to identify biological contexts associated with highly complex gene expression patterns (ChIP-PED: basic and limited gene expression patterns).</p></li>
<li><p>Provides a powerful GUI for users to import gene sets, visualize data, tune parameters, perform cross-species analyses, and export analysis results and plots (ChIP-PED: only command-line tools).</p></li>
</ul>

  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="" id="slide-8" style="background:;">
  <hgroup>
    <h2>GSCA workflow</h2>
  </hgroup>
  <article data-timings="">
    <p><center><img src="fig/workflow.png" width="100%"></center></p>

  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="" id="slide-9" style="background:;">
  <hgroup>
    <h2>GSCA: Conjugate to GSEA</h2>
  </hgroup>
  <article data-timings="">
    <p><center><img src="fig/GSEA.png" width="70%"></center></p>

  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="" id="slide-10" style="background:;">
  <hgroup>
    <h2>GSCA GUI</h2>
  </hgroup>
  <article data-timings="">
    <p><center><img src="fig/GUI.png" width="100%"></center></p>

  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="" id="slide-11" style="background:;">
  <hgroup>
    <h2>Data Packages</h2>
  </hgroup>
  <article data-timings="">
    <p><br></p>

<ul>
<li><p>Affymetrix Human Genome U133a Array (GPL96) : 11778 human samples on 12495 genes</p></li>
<li><p>Affymetrix Mouse Genome 430 2.0 Array (GPL1261) : 9444 mouse samples on 20630 genes</p></li>
<li><p>Affymetrix Human Genome U133 Plus 2.0 Array (GPL570) : 5153 human samples on 19944 genes</p></li>
<li><p>Affymetrix Human Genome U133A 2.0 Array (GPL571) : 313 human samples on 12494 genes</p></li>
</ul>

  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="" id="slide-12" style="background:;">
  <hgroup>
    <h2>Example I: Oct4 High Expression</h2>
  </hgroup>
  <article data-timings="">
    <p><center><img src="fig/Oct4_high.png" width="90%"></center></p>

  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="" id="slide-13" style="background:;">
  <hgroup>
    <h2>Example I: Oct4 Middle Expression</h2>
  </hgroup>
  <article data-timings="">
    <p><center><img src="fig/Oct4_medium.png" width="90%"></center></p>

  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="" id="slide-14" style="background:;">
  <hgroup>
    <h2>Example II: Gli1 and Nanog</h2>
  </hgroup>
  <article data-timings="">
    <p><br></p>

<ul>
<li><p>The Gli proteins are the effetors of Hedgehog signaling and have been shown to be involved in cell fate determination, proliferation and patterning in many cell types and organs during embryo development.</p></li>
<li><p>Nanog is a critical pluripotency marker that is expressed in embryonic stem cells.Neural stem cells also express Nanog and require secreted Hedgehog signaling to maintain proliferation and to regulate differentiation.</p></li>
</ul>

  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="" id="slide-15" style="background:;">
  <hgroup>
    <h2>Example II: Gli1 and Nanog</h2>
  </hgroup>
  <article data-timings="">
    <p><br></p>

<p>Gli1 high expression; Nanog low expression</p>

<p><center><img src="fig/Gli_high_Nanog_low.png" width="105%"></center></p>

  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="" id="slide-16" style="background:;">
  <hgroup>
    <h2>Example II: Gli1 and Nanog</h2>
  </hgroup>
  <article data-timings="">
    <p><br></p>

<p>Gli1 high expression; Nanog high expression</p>

<p><center><img src="fig/Gli_high_Nanog_high.png" width="105%"></center></p>

  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="" id="slide-17" style="background:;">
  <hgroup>
    <h2>Example II: Gli1 and Nanog</h2>
  </hgroup>
  <article data-timings="">
    <p><br></p>

<p>Gli1 high expression; Nanog medium expression</p>

<p><center><img src="fig/Gli_high_Nanog_medium.png" width="105%"></center></p>

  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="" id="slide-18" style="background:;">
  <hgroup>
    <h2>Example III: MYC and Metabolic Pathways</h2>
  </hgroup>
  <article data-timings="">
    <p><br></p>

<p>MYC, mitochondria biogenesis, glycolysis and fatty acid oxidation</p>

<p><center><img src="fig/MYC_interactive.png" width="105%"></center></p>

  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="" id="slide-19" style="background:;">
  <hgroup>
    <h2>Example III: MYC and Metabolic Pathways</h2>
  </hgroup>
  <article data-timings="">
    <p><br></p>

<p>Top ranked biological contexts:</p>

<p><center><img src="fig/MYC_interactive_table.png" width="105%"></center></p>

  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="" id="slide-20" style="background:;">
  <hgroup>
    <h2>Example III: MYC and Metabolic Pathways</h2>
  </hgroup>
  <article data-timings="">
    <p><br></p>

<p><center><img src="fig/MYC_metab_1.png" width="105%"></center></p>

  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="" id="slide-21" style="background:;">
  <hgroup>
    <h2>Example III: MYC and Metabolic Pathways</h2>
  </hgroup>
  <article data-timings="">
    <p><br></p>

<p><center><img src="fig/MYC_metab_2.png" width="105%"></center></p>

  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="" id="slide-22" style="background:;">
  <hgroup>
    <h2>Example III: MYC and Metabolic Pathways</h2>
  </hgroup>
  <article data-timings="">
    <p><br></p>

<p><center><img src="fig/MYC_metab_3.png" width="105%"></center></p>

  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="" id="slide-23" style="background:;">
  <hgroup>
    <h2>Example III: MYC and Metabolic Pathways</h2>
  </hgroup>
  <article data-timings="">
    <p><br></p>

<p><center><img src="fig/MYC_metab_4.png" width="105%"></center></p>

  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="" id="slide-24" style="background:;">
  <hgroup>
    <h2>Install GSCA</h2>
  </hgroup>
  <article data-timings="">
    <ol>
<li><p>Install R </p></li>
<li><p>Install at least one of the four data packages from Bioconductor</p>

<pre><code class="r">  source(&quot;http://bioconductor.org/biocLite.R&quot;)
  biocLite(&quot;Affyhgu133aExpr&quot;)
</code></pre></li>
<li><p>Install GSCA from Github: <a href="https://github.com/zji90/GSCA">https://github.com/zji90/GSCA</a></p>

<pre><code class="r">  if (!require(&quot;devtools&quot;))
    install.packages(&quot;devtools&quot;)
  devtools::install_github(&quot;GSCA&quot;,&quot;zji90&quot;)
</code></pre></li>
<li><p>Launch GSCA GUI:</p>

<pre><code class="r">  library(GSCA)
  GSCAui()
</code></pre></li>
</ol>

  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="" id="slide-25" style="background:;">
  <hgroup>
    <h2>GSCA Resources</h2>
  </hgroup>
  <article data-timings="">
    <p><br></p>

<ul>
<li><p>GSCA github home page: <a href="https://github.com/zji90/GSCA">https://github.com/zji90/GSCA</a></p></li>
<li><p>GSCA online user interface: <a href="http://spark.rstudio.com/jzc19900805/GSCA">http://spark.rstudio.com/jzc19900805/GSCA</a></p></li>
<li><p>GSCA demonstration video: <a href="https://www.youtube.com/watch?v=1OeZ1PAUMhw">https://www.youtube.com/watch?v=1OeZ1PAUMhw</a></p></li>
</ul>

  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="" id="slide-26" style="background:;">
  <hgroup>
    <h2>Demo</h2>
  </hgroup>
  <article data-timings="">
    
  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="segue dark" id="slide-27" style="background:;">
  <hgroup>
    <h2>Thank you!</h2>
  </hgroup>
  <article data-timings="">
    
  </article>
  <!-- Presenter Notes -->
</slide>

    <slide class="backdrop"></slide>
  </slides>
  <div class="pagination pagination-small" id='io2012-ptoc' style="display:none;">
    <ul>
      <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=1 title='GSCA Overview'>
         1
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=2 title='Background'>
         2
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=3 title='Challenges'>
         3
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=4 title='ChIP-PED'>
         4
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=5 title='ChIP-PED'>
         5
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=6 title='ChIP-PED Limitations'>
         6
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=7 title='GSCA'>
         7
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=8 title='GSCA workflow'>
         8
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=9 title='GSCA: Conjugate to GSEA'>
         9
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=10 title='GSCA GUI'>
         10
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=11 title='Data Packages'>
         11
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=12 title='Example I: Oct4 High Expression'>
         12
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=13 title='Example I: Oct4 Middle Expression'>
         13
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=14 title='Example II: Gli1 and Nanog'>
         14
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=15 title='Example II: Gli1 and Nanog'>
         15
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=16 title='Example II: Gli1 and Nanog'>
         16
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=17 title='Example II: Gli1 and Nanog'>
         17
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=18 title='Example III: MYC and Metabolic Pathways'>
         18
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=19 title='Example III: MYC and Metabolic Pathways'>
         19
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=20 title='Example III: MYC and Metabolic Pathways'>
         20
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=21 title='Example III: MYC and Metabolic Pathways'>
         21
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=22 title='Example III: MYC and Metabolic Pathways'>
         22
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=23 title='Example III: MYC and Metabolic Pathways'>
         23
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=24 title='Install GSCA'>
         24
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=25 title='GSCA Resources'>
         25
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=26 title='Demo'>
         26
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=27 title='Thank you!'>
         27
      </a>
    </li>
  </ul>
  </div>  <!--[if IE]>
    <script 
      src="http://ajax.googleapis.com/ajax/libs/chrome-frame/1/CFInstall.min.js">  
    </script>
    <script>CFInstall.check({mode: 'overlay'});</script>
  <![endif]-->
</body>
  <!-- Load Javascripts for Widgets -->
  
  <!-- MathJax: Fall back to local if CDN offline but local image fonts are not supported (saves >100MB) -->
  <script type="text/x-mathjax-config">
    MathJax.Hub.Config({
      tex2jax: {
        inlineMath: [['$','$'], ['\\(','\\)']],
        processEscapes: true
      }
    });
  </script>
  <script type="text/javascript" src="http://cdn.mathjax.org/mathjax/2.0-latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>
  <!-- <script src="https://c328740.ssl.cf1.rackcdn.com/mathjax/2.0-latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML">
  </script> -->
  <script>window.MathJax || document.write('<script type="text/x-mathjax-config">MathJax.Hub.Config({"HTML-CSS":{imageFont:null}});<\/script><script src="libraries/widgets/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"><\/script>')
</script>
<!-- LOAD HIGHLIGHTER JS FILES -->
  <script src="libraries/highlighters/highlight.js/highlight.pack.js"></script>
  <script>hljs.initHighlightingOnLoad();</script>
  <!-- DONE LOADING HIGHLIGHTER JS FILES -->
   
  </html>