<!DOCTYPE html>
<html>
<head>
  <title>TSCAN: Tools for Single-Cell ANalysis</title>
  <meta charset="utf-8">
  <meta name="description" content="TSCAN: Tools for Single-Cell ANalysis">
  <meta name="author" content="Zhicheng Ji">
  <meta name="generator" content="slidify" />
  <meta name="apple-mobile-web-app-capable" content="yes">
  <meta http-equiv="X-UA-Compatible" content="chrome=1">
  <link rel="stylesheet" href="libraries/frameworks/io2012/css/default.css" media="all" >
  <link rel="stylesheet" href="libraries/frameworks/io2012/css/phone.css" 
    media="only screen and (max-device-width: 480px)" >
  <link rel="stylesheet" href="libraries/frameworks/io2012/css/slidify.css" >
  <link rel="stylesheet" href="libraries/highlighters/highlight.js/css/tomorrow.css" />
  <base target="_blank"> <!-- This amazingness opens all links in a new tab. -->  <link rel=stylesheet href="./assets/css/ribbons.css"></link>

  
  <!-- Grab CDN jQuery, fall back to local if offline -->
  <script src="http://ajax.aspnetcdn.com/ajax/jQuery/jquery-1.7.min.js"></script>
  <script>window.jQuery || document.write('<script src="libraries/widgets/quiz/js/jquery.js"><\/script>')</script> 
  <script data-main="libraries/frameworks/io2012/js/slides" 
    src="libraries/frameworks/io2012/js/require-1.0.8.min.js">
  </script>
  
  

</head>
<body style="opacity: 0">
  <slides class="layout-widescreen">
    
    <!-- LOGO SLIDE -->
        <slide class="title-slide segue nobackground">
  <aside class="gdbar">
    <img src="assets/img/bloomberg_shield.png">
  </aside>
  <hgroup class="auto-fadein">
    <h1>TSCAN: Tools for Single-Cell ANalysis</h1>
    <h2>A Noval Method for Single-Cell RNA-seq Pseudotemporal Ordering</h2>
    <p>Zhicheng Ji<br/>Lab Meeting Oct 3rd</p>
  </hgroup>
  <article></article>  
</slide>
    

    <!-- SLIDES -->
    <slide class="" id="slide-1" style="background:;">
  <hgroup>
    <h2>Introductions</h2>
  </hgroup>
  <article data-timings="">
    <p>Single-cell RNA-seq is a powerful approach to investigate the heterogeneity of gene expression activities on cell level, which is otherwise hard to detect by bulk transcriptomics experiments.</p>

<p><center><img src="fig/singlecell_RNA-Seq_workflow.jpg" width="70%"></center></p>

  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="" id="slide-2" style="background:;">
  <hgroup>
    <h2>Introductions</h2>
  </hgroup>
  <article data-timings="">
    <p>Advantages of Single-cell RNA-seq compared to bulk RNA-seq:</p>

<p>Avoid potential Simpson&#39;s paradox:</p>

<p><center><img src="fig/Simpson's_paradox.png" width="70%"></center></p>

  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="" id="slide-3" style="background:;">
  <hgroup>
    <h2>Introductions</h2>
  </hgroup>
  <article data-timings="">
    <p>Advantages of Single-cell RNA-seq compared to bulk RNA-seq:</p>

<p>Increase temporal resolution:</p>

<p><center><img src="fig/temporal_resolution.png" width="100%"></center></p>

  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="" id="slide-4" style="background:;">
  <hgroup>
    <h2>Introductions</h2>
  </hgroup>
  <article data-timings="">
    <p>Many single-cell RNA-seq data involve a continuous biological process, which requires computational methods to realign the cells to a pseudotime scale.</p>

<p>Current computational approaches:</p>

<ul>
<li><p>SPADES: unsupervised spanning-tree progression approach to organize cells in a hierarchy of related phenotypes when applied to flow cytometry data.</p></li>
<li><p>Monocle: unsupervised approach to order single cells by constructing a minimum-spanning-tree. Capable of finding different trajectories of biological process. </p></li>
</ul>

  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="" id="slide-5" style="background:;">
  <hgroup>
    <h2>Introductions</h2>
  </hgroup>
  <article data-timings="">
    <p>Disadvantages of current methods:</p>

<ul>
<li><p>Unsupervised methods cannot incorporate prior biological knowledge (e.g. expressions of marker genes).</p></li>
<li><p>Users have limited options to tune the pseudotemporal orderings.</p></li>
<li><p>No quantitative methods to evaluate different pseudotemporal orderings.</p></li>
</ul>

  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="" id="slide-6" style="background:;">
  <hgroup>
    <h2>Introductions</h2>
  </hgroup>
  <article data-timings="">
    <p>TSCAN, a novel method &amp; software package to address these problems:</p>

<ul>
<li><p>Can switch from unsupervised to supervised method to incorporate prior biological knowledge.</p></li>
<li><p>Plenty of options for users to conveniently tune the pseudotemporal orderings.</p></li>
<li><p>POS: quantitative methods to evaluate pseudotemporal orderings.</p></li>
</ul>

  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="" id="slide-7" style="background:;">
  <hgroup>
    <h2>Software Package</h2>
  </hgroup>
  <article data-timings="">
    <p>TSCAN is already available on Bioconductor:</p>

<p><a href="http://www.bioconductor.org/packages/devel/bioc/html/TSCAN.html">http://www.bioconductor.org/packages/devel/bioc/html/TSCAN.html</a></p>

<p>TSCAN GUI can be directly launched on:</p>

<p><a href="https://zhiji.shinyapps.io/TSCAN">https://zhiji.shinyapps.io/TSCAN</a></p>

  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="" id="slide-8" style="background:;">
  <hgroup>
    <h2>Datasets</h2>
  </hgroup>
  <article data-timings="">
    <ul>
<li><p>Human skeletal muscle myoblasts (HSMM): 4 time points; 271 cells; 47192 genes</p></li>
<li><p>Mouse LPS stimulation (pathogen): 5 time points; 363 cells; 27723 genes</p></li>
</ul>

  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="" id="slide-9" style="background:;">
  <hgroup>
    <h2>POS</h2>
  </hgroup>
  <article data-timings="">
    <p>Pseudotemporal Ordering Score: quantitatively measure different pseudotemporal orderings.</p>

<p>Requires prior knowledge of at least two cell sub-populations.</p>

<p>Cells collected at later time points should also be assigned to a later position on pseudotime course.</p>

<p>Definition:</p>

<p>\(\frac{\sum_{i,j:\pi(i) \prec \pi(j)} (T_j-T_i)}{N_1*N_2}\)</p>

  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="" id="slide-10" style="background:;">
  <hgroup>
    <h2>TSCAN Methods</h2>
  </hgroup>
  <article data-timings="">
    <ol>
<li><p>Add pseudocount 1 and take log2 of data.</p></li>
<li><p>Select key genes: expression greater than 0.5 in at least 30% of all cells. CV in non-dropout cell greater than the median CV. (HSMM: 3989 genes; LPS: 2892 genes)</p></li>
<li><p>Perform principal component analysis with selected optimal dimension.</p></li>
<li><p>Multivariate normal clustering using the data after PCA.</p></li>
<li><p>Connect all cluster centers with minimum-spanning-tree.</p></li>
<li><p>Assign all points on the time course by projections.</p></li>
</ol>

  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="" id="slide-11" style="background:;">
  <hgroup>
    <h2>HSMM Results</h2>
  </hgroup>
  <article data-timings="">
    <p>Monocle :</p>

<p><center><img src="fig/HSMM_Monocle.png"></center></p>

  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="" id="slide-12" style="background:;">
  <hgroup>
    <h2>HSMM Results</h2>
  </hgroup>
  <article data-timings="">
    <p>MST after mclust:</p>

<p><center><img src="fig/HSMM_mclusttree.png"></center></p>

  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="" id="slide-13" style="background:;">
  <hgroup>
    <h2>HSMM Results</h2>
  </hgroup>
  <article data-timings="">
    <p>Monocle POS: 0.4457489</p>

<p>TSCAN POS: 0.5268724</p>

  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="" id="slide-14" style="background:;">
  <hgroup>
    <h2>HSMM Results</h2>
  </hgroup>
  <article data-timings="">
    <p>CDK1:</p>

<p><center><img src="fig/HSMM_CDK1.png"></center></p>

  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="" id="slide-15" style="background:;">
  <hgroup>
    <h2>HSMM Results</h2>
  </hgroup>
  <article data-timings="">
    <p>ID1:</p>

<p><center><img src="fig/HSMM_ID1.png"></center></p>

  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="" id="slide-16" style="background:;">
  <hgroup>
    <h2>HSMM Results</h2>
  </hgroup>
  <article data-timings="">
    <p>MYOG:</p>

<p><center><img src="fig/HSMM_MYOG.png"></center></p>

  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="" id="slide-17" style="background:;">
  <hgroup>
    <h2>HSMM Results</h2>
  </hgroup>
  <article data-timings="">
    <p>MEF2C:</p>

<p><center><img src="fig/HSMM_MEF2C.png"></center></p>

  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="" id="slide-18" style="background:;">
  <hgroup>
    <h2>HSMM Results</h2>
  </hgroup>
  <article data-timings="">
    <p>MYH2:</p>

<p><center><img src="fig/HSMM_MYH2.png"></center></p>

  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="" id="slide-19" style="background:;">
  <hgroup>
    <h2>LPS Results</h2>
  </hgroup>
  <article data-timings="">
    <p>Monocle :</p>

<p><center><img src="fig/LPS_Monocle.png"></center></p>

  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="" id="slide-20" style="background:;">
  <hgroup>
    <h2>LPS Results</h2>
  </hgroup>
  <article data-timings="">
    <p>MST after mclust:</p>

<p><center><img src="fig/LPS_mclusttree.png"></center></p>

  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="" id="slide-21" style="background:;">
  <hgroup>
    <h2>LPS Results</h2>
  </hgroup>
  <article data-timings="">
    <p>Monocle POS: 0.4279521</p>

<p>TSCAN POS: 0.7135031</p>

<p>TSCAN STAT2 adjusted POS: 0.7620408 </p>

  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="" id="slide-22" style="background:;">
  <hgroup>
    <h2>LPS Results</h2>
  </hgroup>
  <article data-timings="">
    <p>STAT1:</p>

<p><center><img src="fig/LPS_STAT1.png" width="100%"></center></p>

  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="" id="slide-23" style="background:;">
  <hgroup>
    <h2>LPS Results</h2>
  </hgroup>
  <article data-timings="">
    <p>STAT2:</p>

<p><center><img src="fig/LPS_STAT2.png" width="100%"></center></p>

  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="" id="slide-24" style="background:;">
  <hgroup>
    <h2>LPS Results</h2>
  </hgroup>
  <article data-timings="">
    <p>NFKB1:</p>

<p><center><img src="fig/LPS_NFKB1.png" width="100%"></center></p>

  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="" id="slide-25" style="background:;">
  <hgroup>
    <h2>LPS Results</h2>
  </hgroup>
  <article data-timings="">
    <p>IRF9:</p>

<p><center><img src="fig/LPS_IRF9.png" width="100%"></center></p>

  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="" id="slide-26" style="background:;">
  <hgroup>
    <h2>Future work</h2>
  </hgroup>
  <article data-timings="">
    <ul>
<li><p>TSCAN 2.0: Tools for time course Single-Cell RNA-seq data</p></li>
<li><p>TSCAN 2.1: Tools for differential gene expression in Single-Cell RNA-seq data</p></li>
<li><p>TSCAN 3.0: Tools for alternative splicing analysis in Single-Cell RNA-seq data</p></li>
<li><p>GSCA 2.0: Similarity and time course analysis for PED</p></li>
</ul>

  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="segue dark" id="slide-27" style="background:;">
  <hgroup>
    <h2>Thank you!</h2>
  </hgroup>
  <article data-timings="">
    
  </article>
  <!-- Presenter Notes -->
</slide>

    <slide class="backdrop"></slide>
  </slides>
  <div class="pagination pagination-small" id='io2012-ptoc' style="display:none;">
    <ul>
      <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=1 title='Introductions'>
         1
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=2 title='Introductions'>
         2
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=3 title='Introductions'>
         3
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=4 title='Introductions'>
         4
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=5 title='Introductions'>
         5
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=6 title='Introductions'>
         6
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=7 title='Software Package'>
         7
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=8 title='Datasets'>
         8
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=9 title='POS'>
         9
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=10 title='TSCAN Methods'>
         10
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=11 title='HSMM Results'>
         11
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=12 title='HSMM Results'>
         12
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=13 title='HSMM Results'>
         13
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=14 title='HSMM Results'>
         14
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=15 title='HSMM Results'>
         15
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=16 title='HSMM Results'>
         16
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=17 title='HSMM Results'>
         17
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=18 title='HSMM Results'>
         18
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=19 title='LPS Results'>
         19
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=20 title='LPS Results'>
         20
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=21 title='LPS Results'>
         21
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=22 title='LPS Results'>
         22
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=23 title='LPS Results'>
         23
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=24 title='LPS Results'>
         24
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=25 title='LPS Results'>
         25
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=26 title='Future work'>
         26
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=27 title='Thank you!'>
         27
      </a>
    </li>
  </ul>
  </div>  <!--[if IE]>
    <script 
      src="http://ajax.googleapis.com/ajax/libs/chrome-frame/1/CFInstall.min.js">  
    </script>
    <script>CFInstall.check({mode: 'overlay'});</script>
  <![endif]-->
</body>
  <!-- Load Javascripts for Widgets -->
  
  <!-- MathJax: Fall back to local if CDN offline but local image fonts are not supported (saves >100MB) -->
  <script type="text/x-mathjax-config">
    MathJax.Hub.Config({
      tex2jax: {
        inlineMath: [['$','$'], ['\\(','\\)']],
        processEscapes: true
      }
    });
  </script>
  <script type="text/javascript" src="http://cdn.mathjax.org/mathjax/2.0-latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>
  <!-- <script src="https://c328740.ssl.cf1.rackcdn.com/mathjax/2.0-latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML">
  </script> -->
  <script>window.MathJax || document.write('<script type="text/x-mathjax-config">MathJax.Hub.Config({"HTML-CSS":{imageFont:null}});<\/script><script src="libraries/widgets/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"><\/script>')
</script>
<!-- LOAD HIGHLIGHTER JS FILES -->
  <script src="libraries/highlighters/highlight.js/highlight.pack.js"></script>
  <script>hljs.initHighlightingOnLoad();</script>
  <!-- DONE LOADING HIGHLIGHTER JS FILES -->
   
  </html>