---
title       : "GSCA: Gene Set Context Analysis"
subtitle    : "A Software Tool for Turning Publicly Available Gene Expression Data into Discoveries"
author      : Zhicheng Ji, Hongkai Ji
job         : 
logo        : bloomberg_shield.png
framework   : io2012        # {io2012, html5slides, shower, dzslides, ...}
highlighter : highlight.js  # {highlight.js, prettify, highlight}
hitheme     : tomorrow
widgets     : [mathjax]            # {mathjax, quiz, bootstrap}
mode        : selfcontained # {standalone, draft}
github      :
      user  : zji90
      repo  : GSCA-presentation

---

## GSCA Overview

<br>

* An open source software package with a powerful Graphical User Interface (GUI).

* Interactively visualize and explore gene and gene set activities in 25,000+ human and mouse biological contexts.

* Systematically identify biological contexts associated with specific gene set activity patterns. 

* Discover previously unknown contexts of gene set functions.

---

## Background

* 200,000+ gene expression samples stored in public databases such as GEO and ArrayExpress.

* PED contain vast amount of information, but not fully utilized so far.

<!-- Microarray dataset 1 generated by one investigator for studying pathway 2 may also contain information about pathway 1. This information may not be used by the original investigator for his/her study of pathway 2, but it can be useful for other people who want to study pathway 1 --> 

<center><img src="fig/background.png" width="70%"></center>

---

## Challenges

<br>

* Reuse of PED enables one to systematically examine gene or pathwayâ€™s activities in a broad spectrum of biological contexts, saving resources of generating the data.

* Several roadblocks to be removed before PED can be reused:

      + Gene expression data should be constantly normalized and annotated.

      + Data should be easy to visualize and retrieve.

      + One needs efficient way to analyze the data and turn them into useful knowledge.

* None of these is trivial given the complexity, heterogeneity and size of the data.

---

## ChIP-PED

* Analyzes the expression of a transcription factor (TF) and its target genes in PED to find biological contexts associated with the function of the TF.

* Myc: a regulator gene that codes for a transcription factor.

<center><img src="fig/chipped1.png" width="75%"></center>

---

## ChIP-PED

* Knockdown of MYC: substantially slower proliferation rate and decreased tumorigenicity.

<center><img src="fig/chipped2.png" width="100%"></center>

---

## ChIP-PED Limitations

<br>

* Can only analyze one TF and its target genes.

* The gene expression pattern is limited (TF+TG+; TF+TG-; TF-TG+; TF-TG-)

* Only command-lines. Hard to use.

---

## GSCA

<br>

* Built upon 25,000+ pre-normalized and expert-annotated human and mouse gene expression samples with 1000+ biological contexts.

* Takes any number of gene sets of any kind as inputs (ChIP-PED: one TF and its target genes).

* Allows users to identify biological contexts associated with highly complex gene expression patterns (ChIP-PED: basic and limited gene expression patterns).

* Provides a powerful GUI for users to import gene sets, visualize data, tune parameters, perform cross-species analyses, and export analysis results and plots (ChIP-PED: only command-line tools).

---

## GSCA workflow

<center><img src="fig/workflow.png" width="100%"></center>

---

## GSCA: Conjugate to GSEA

<center><img src="fig/GSEA.png" width="70%"></center>

---

## GSCA GUI

<center><img src="fig/GUI.png" width="100%"></center>

---

## Data Packages

<br>

* Affymetrix Human Genome U133a Array (GPL96) : 11778 human samples on 12495 genes

* Affymetrix Mouse Genome 430 2.0 Array (GPL1261) : 9444 mouse samples on 20630 genes

* Affymetrix Human Genome U133 Plus 2.0 Array (GPL570) : 5153 human samples on 19944 genes

* Affymetrix Human Genome U133A 2.0 Array (GPL571) : 313 human samples on 12494 genes

---

## Example I: Oct4 High Expression

<center><img src="fig/Oct4_high.png" width="90%"></center>

---

## Example I: Oct4 Middle Expression

<center><img src="fig/Oct4_medium.png" width="90%"></center>

---

## Example II: Gli1 and Nanog

<br>

* The Gli proteins are the effetors of Hedgehog signaling and have been shown to be involved in cell fate determination, proliferation and patterning in many cell types and organs during embryo development.

* Nanog is a critical pluripotency marker that is expressed in embryonic stem cells.Neural stem cells also express Nanog and require secreted Hedgehog signaling to maintain proliferation and to regulate differentiation.

---

## Example II: Gli1 and Nanog

<br>

Gli1 high expression; Nanog low expression

<center><img src="fig/Gli_high_Nanog_low.png" width="105%"></center>


---

## Example II: Gli1 and Nanog

<br>

Gli1 high expression; Nanog high expression

<center><img src="fig/Gli_high_Nanog_high.png" width="105%"></center>

---

## Example II: Gli1 and Nanog

<br>

Gli1 high expression; Nanog medium expression

<center><img src="fig/Gli_high_Nanog_medium.png" width="105%"></center>

---

## Example III: MYC and Metabolic Pathways 

<br>

MYC, mitochondria biogenesis, glycolysis and fatty acid oxidation

<center><img src="fig/MYC_interactive.png" width="105%"></center>

---

## Example III: MYC and Metabolic Pathways 

<br>

Top ranked biological contexts:

<center><img src="fig/MYC_interactive_table.png" width="105%"></center>

---

## Example III: MYC and Metabolic Pathways 

<br>

<center><img src="fig/MYC_metab_1.png" width="105%"></center>

---

## Example III: MYC and Metabolic Pathways 

<br>

<center><img src="fig/MYC_metab_2.png" width="105%"></center>

---

## Example III: MYC and Metabolic Pathways 

<br>

<center><img src="fig/MYC_metab_3.png" width="105%"></center>

---

## Example III: MYC and Metabolic Pathways 

<br>

<center><img src="fig/MYC_metab_4.png" width="105%"></center>

---

## Install GSCA

1. Install R 

2. Install at least one of the four data packages from Bioconductor

      
      ```r
      source("http://bioconductor.org/biocLite.R")
      biocLite("Affyhgu133aExpr")
      ```

3. Install GSCA from Github: https://github.com/zji90/GSCA

      
      ```r
      if (!require("devtools"))
        install.packages("devtools")
      devtools::install_github("GSCA","zji90")
      ```
      
4. Launch GSCA GUI:

      
      ```r
      library(GSCA)
      GSCAui()
      ```
      

---

## GSCA Resources

<br>

* GSCA github home page: https://github.com/zji90/GSCA

* GSCA online user interface: http://spark.rstudio.com/jzc19900805/GSCA

* GSCA demonstration video: https://www.youtube.com/watch?v=1OeZ1PAUMhw

---

## Demo



--- .segue .dark

Thank you!
-----



